turbonfs/inc/nfs_internal.h

```diff
@@ -108,7 +108,8 @@ struct mount_options
         actimeo(aznfsc_cfg.actimeo),
         readdir_maxcount(aznfsc_cfg.readdir_maxcount),
         readahead_kb(aznfsc_cfg.readahead_kb),
-        auth(aznfsc_cfg.auth)
+        auth(aznfsc_cfg.auth),
+        authtype(aznfsc_cfg.authtype)
```

Comment:
Why we need to set authType as aznfsc_cfg has default value.
We can set in constructor according to auth
authtype = AzAuthNone;
If (auth)
 authtype = AzAuthAAD;